feat(Database,Table): support fetch repetition field value

qinqinghai opened the pull request.

DASPRiD commented:
See failed commitlint test. Also please rebase against main – we had some issue in our CI so code coverage reports did not run.

qinqinghai committed:
feat(Database): add repeating field type to metadata

add GenericCollectionFieldMetadata to handle '@MaxRepetitions', $Type 'Collection(Edm.String)' ...

qinqinghai committed:
feat(Table): add function to fetch repetition field value

qinqinghai committed:
test(Table): add test for fetch repetition field value

qinqinghai commented:
thanks
